python/ray/serve/backend_state.py

```diff
@@ -606,26 +570,82 @@ def __init__(self, name: str, controller_name: str, detached: bool,
         self._target_replicas: int = -1
         self._curr_goal: Optional[GoalId] = None
         self._target_version: BackendVersion = None
-        self._prev_startup_warning: float = -1
+        self._prev_startup_warning: float = time.time()
```

Comment:
Hmmm I'm pretty sure I had to do this for a test to work at some point... if the unit tests are passing though, no problem